packages/bigtable/test/index.js

```diff
@@ -172,10 +172,19 @@ describe('Bigtable', function() {
       ]);
 
       assert.deepEqual(calledWith.packageJson, require('../package.json'));
+
+      Object.keys(calledWith.protoServices).forEach(function(serviceKey) {
```

Comment:
We can skip this test here, because it's not code that was done in `bigtable/src/index.js`.